projects/opencv/imread_fuzzer.cc

```diff
@@ -0,0 +1,69 @@
+#include <cstddef>
+#include <cstdint>
+#include <unistd.h>
+
+#include <opencv2/opencv.hpp>
+
+namespace {
+
+static char* fuzzer_get_tmpfile(const uint8_t* data, size_t size) {
```

Comment:
Would it make sense to keep this boilerplate in a separate file? The libxml2 fuzzers have a copy of these in fuzzer_temp_file.h. If we have the same file copied everywhere it might be easier to factor things out in a future refactoring.

Reply:
Good point, I didn't realize this was already in OSS-Fuzz. The Dockerfile doesn't support reading files outside the project's directory. Maybe we could put it in the base builder image somewhere?

Reply:
My thinking was that we could clone this file in every place it is needed, and hopefully factor things out if we get too many copies. But maybe others have better ideas.

Reply:
ah ok, so just copy the .h file into the each project directory as needed? Then we can refactor it out later easily? That sounds fine to me. Does anyone from OSS-Fuzz have an opinion?

Reply:
+1 to this idea.

Reply:
Done, PTAL.